genai-perf/genai_perf/inputs/converters/tensorrtllm_engine_converter.py

```diff
@@ -25,7 +25,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
Small plug: [hyperfine](https://github.com/sharkdp/hyperfine) is a nifty command line tool for measuring average speed of shell commands.

Reply:
Thanks for the suggestion!